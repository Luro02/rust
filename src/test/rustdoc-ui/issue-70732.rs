// compile-flags:--test --test-args --test-threads=1
// normalize-stdout-test: "src/test/rustdoc-ui" -> "$$DIR"
//
// The indentation would be stripped from a doc attribute in between normal doc comments
// (`//!` or `///`).
//
// https://github.com/rust-lang/rust/issues/70732

//! ```rust
//! let x = r#"
//!   bar();
//! "#;
//! assert_eq!(x, "\n  bar();\n");
//! ```

#![doc = " ```rust"]
#![doc = " let x = r#\""]
#![doc = "   bar();"]
#![doc = " \"#;"]
#![doc = " assert_eq!(x, \"\\n  bar();\\n\");"]
#![doc = " ```"]

//! ```rust
//! let x = r#"
#![doc = "   bar();"]
//! "#;
//! assert_eq!(x, "\n  bar();\n");
//! ```
