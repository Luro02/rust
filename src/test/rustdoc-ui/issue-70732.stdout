
running 3 tests
test $DIR/issue-70732.rs -  (line 15) ... ok
test $DIR/issue-70732.rs -  (line 21) ... ok
test $DIR/issue-70732.rs -  (line 9) ... ok

test result: ok. 3 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
